tezos/new_context/src/tezedge_context.rs

```diff
@@ -42,7 +43,7 @@ pub struct PatchContextFunction {}
 pub struct TezedgeIndex {
     pub repository: Arc<RwLock<ContextKeyValueStore>>,
     pub patch_context: Rc<Option<BoxRoot<PatchContextFunction>>>,
-    pub strings: Rc<RefCell<StringInterner>>,
+    pub storage: Rc<RefCell<Storage>>,
```

Comment:
Adding comments for this struct briefly explaining repository and storage would be good.